src/usher_mapper.cpp

```diff
@@ -392,7 +395,7 @@ void mapper2_body(mapper2_input& input, bool compute_parsimony_scores) {
                 }
             }
         }
-        if (found) {
+        if (found_pos) {
```

Comment:
A back mutation is probably not necessary if a nucleotide mutated in the ancestors of the new sample and also in the new sample itself, as it would have been accounted after line 334.

Reply:
I think you're right about this.